glide.lock

```diff
@@ -250,9 +256,11 @@ imports:
   - internal/bitbucket.org/ww/goautoneg
   - model
 - name: github.com/prometheus/procfs
-  version: 1878d9fbb537119d24b21ca07effd591627cd160
+  version: 3c6d1cd72106fb2c83336c7766b365da0cf9574e
+  subpackages:
+  - xfs
 - name: github.com/Sirupsen/logrus
-  version: 55eb11d21d2a31a3cc93838241d04800f52e823d
+  version: 3f603f494d61c73457fb234161d8982b9f0f0b71
```

Comment:
when I run the cleanall and install-deps rules, this change is reverted, did you use the same rules to update the glide.lock? (in particular the cleanall rule)